stl/inc/flat_set

```diff
@@ -164,6 +183,7 @@ public:
     _NODISCARD const_reverse_iterator rend() const noexcept {

```

Comment:
For `rbegin()`, I think it's not necessary to call `_Get_cont().rend()`, as it is required to be `reverse_iterator(begin())`, and the container does not necessarily offer `rbegin()` method (am I mistaken?). The same goes for `rend(), crbegin() and crend()`.

Reply:
> the container does not necessarily offer `rbegin()` method

No, `rbegin` must be provided since `flat_set`/`flat_multiset` requires the adapted container to be a reversible container ([[flat.set.overview]/2](https://eel.is/c++draft/flat.set.overview#2), [[flat.multiset.overview]/2](https://eel.is/c++draft/flat.multiset.overview#2)).

Reply:
However, that's the requirements for `flat_meow`; the overview also says `"Any sequence container supporting Cpp17RandomAccessIterator can be used to instantiate flat_meow"` ([ref](https://eel.is/c++draft/flat.multiset.overview#7))

Reply:
Oh. I was just confused. The requirement on reversible containers is always additionally specified and not implied by other requirements.